src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lotteryBall = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
Consider defining a constant for the number of iterations (3) in the loop. This will improve code readability and maintainability by avoiding the use of magic numbers. For example, you could define a constant `private static final int NUMBER_OF_BALLS = 3;` and use it in the loop condition.